src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor/RazorWrapperFactory.cs

```diff
@@ -58,26 +58,6 @@ private static ImmutableArray<TResult> WrapAll<TInner, TResult>(ImmutableArray<T
         return builder.DrainToImmutable();
     }
 
-    private static ImmutableArray<TResult> WrapAll<TInner, TResult>(IReadOnlyList<TInner> list, Func<TInner, TResult> createWrapper)
-        where TInner : class
-        where TResult : class
-    {
-        var count = list.Count;
-        if (count == 0)
-        {
-            return ImmutableArray<TResult>.Empty;
-        }
-
-        using var builder = new PooledArrayBuilder<TResult>(capacity: count);
-
-        for (var i = 0; i < count; i++)
-        {
-            builder.Add(createWrapper(list[i]));
-        }
-
-        return builder.DrainToImmutable();
-    }
-
```

Comment:
ℹ️ After converting `RazorCSharpDocument` to an `ImmutableArray<RazorDiagnostic>`, this method is no longer used.